core/README.md

```diff
@@ -0,0 +1,4 @@
+This package is legacy and no longer developed.
+
+Users are encouraged to look into more actively developed test frameworks,
+e.g. [tasty](https://hackage.haskell.or/package/tasty).
```

Comment:
@andreasabel I think we discussed this before https://github.com/haskell/test-framework/issues/62